app/scripts/migrations/092.3.ts

```diff
@@ -0,0 +1,52 @@
+import { isNullOrUndefined } from '@metamask/utils';
+import { cloneDeep } from 'lodash';
+
+type VersionedData = {
+  meta: { version: number };
+  data: Record<string, unknown>;
+};
+
+export const version = 92.3;
+
+/**
+ * This migration does the following:
+ *
+ * - Deletes currently stored advancedGasFee in preferences controller, replacing the default with an empty object
```

Comment:
Nit: We should explain the `hadAdvancedGasFeesSetPriorToMigration92_3` flag here as well

Reply:
done